kong/router/traditional.lua

```diff
@@ -1448,6 +1450,20 @@ function _M.new(routes, cache, cache_neg)
   local match_sources        = not isempty(plain_indexes.sources)
   local match_destinations   = not isempty(plain_indexes.destinations)
 
+  -- warning about the regex cache size being too small
+  if not lua_regex_cache_max_entries then
+    lua_regex_cache_max_entries = kong.configuration.nginx_http_lua_regex_cache_max_entries or 1024
```

Comment:
Should we default here to 8192 too?

Reply:
This seems non-sense because OpenResty's default value of `lua_regex_cache_max_entries` is `1024` (https://github.com/openresty/lua-nginx-module/blob/master/README.markdown#lua_regex_cache_max_entries), we set the `nginx_http_lua_regex_cache_max_entries` to `8192`, but the user could overwrite it.